### PR TITLE
feat: add core message payload validation

### PR DESCRIPTION
**Pull request by jonaslagoni**
**Description**
This was initially added to v3 here: https://github.com/asyncapi/spec-json-schemas/pull/334

This PR replicates it for v2.

### CODE REVIEW

File: definitions/2.0.0/message.json [hunk 1]
```diff
@@ -119,7 +119,70 @@
...
+              "if": {
+                "required": [
+                  "schemaFormat"
+                ],
+                "properties": {
+                  "schemaFormat": {
+                    "enum": [
+                      "application/vnd.aai.asyncapi;version=2.0.0",
+                      "application/vnd.aai.asyncapi+json;version=2.0.0",
+                      "application/vnd.aai.asyncapi+yaml;version=2.0.0"
```
Comment (jonaslagoni) [inline definitions/2.0.0/message.json:150]: Should v2.0.0 add support for up until v2.6.0? 🤔

**Review by dalelane — APPROVED** [review]

**Review by smoya — APPROVED** [review]
LGTM! 🚀🌔

**Comment by jonaslagoni** [comment]
Thanks for the reviews folks 🎉

**Pull request merged**